tests/unit/challenges/test_views.py

```diff
@@ -99,8 +101,9 @@ def setUp(self):
             approved_by_admin=False,
         )
 
-        self.url = reverse_lazy('challenges:get_challenge_list',
-                                kwargs={'challenge_host_team_pk': self.challenge_host_team.pk})
+        self.url = reverse_lazy(
```

Comment:
I think your editor has made a lot of unnecessary changes which are not needed in this PR. I can't review the changes since there is a lot of irrelevant changes. Can you please remove it.